test/tests.ml

```diff
@@ -31,15 +31,33 @@ let wait pid =
   | WSIGNALED n -> Printf.ksprintf failwith "got signal %d" n
   | WSTOPPED _ -> assert false
 
+let list_files = Filename.concat (Sys.getcwd ()) "exe/list_files.exe"
+
+let () =
+  Unix.mkdir "sub" 0o777;
+  close_out (open_out "sub/foo");
+  close_out (open_out "sub/bar")
+
+let%expect_test "cwd:Path" =
+  wait
+    (Spawn.spawn () ~prog:list_files ~argv:[ "list_files.exe" ]
+       ~cwd:(Path "sub"));
+  [%expect {|
+    bar
+    foo
+  |}]
+
 let%expect_test "cwd:Fd" =
   (if Sys.win32 then
-    print_string "/tmp"
+    print_endline "foo"
```

Comment:
Shouldn't we print two lines here ("bar", then "foo"), instead of just foo?

Reply:
Indeed, good catch. I didn't even look at the Windows CI as it is currently broken for other reasons. Need to unbreak it.